diff --git a/node_modules/@webpack-cli/serve/lib/index.js b/node_modules/@webpack-cli/serve/lib/index.js
index 5352ee6..cb865bc 100644
--- a/node_modules/@webpack-cli/serve/lib/index.js
+++ b/node_modules/@webpack-cli/serve/lib/index.js
@@ -29,6 +29,11 @@ function serve(...args) {
     if (webpackArgs && webpackArgs._all && typeof webpackArgs._all.hot !== 'undefined') {
         finalArgs['hot'] = webpackArgs._all.hot;
     }
+
+    const conf = require(require('path').join(__dirname, '..', '..', '..', '..', 'webpack.devServer.config.js'))
+    console.log('devConf', conf)
+    Object.assign(finalArgs, conf)
+
     cli.getCompiler(webpackArgs, core).then((compiler) => {
         startDevServer_1.default(compiler, finalArgs);
     });
