diff --git a/node_modules/proxy-state-tree/es/Proxyfier.js b/node_modules/proxy-state-tree/es/Proxyfier.js
index 1c3ad73..55ef042 100644
--- a/node_modules/proxy-state-tree/es/Proxyfier.js
+++ b/node_modules/proxy-state-tree/es/Proxyfier.js
@@ -231,7 +231,7 @@ export class Proxifier {
                     return proxifier.tree;
                 if (typeof prop === 'symbol' || prop in Object.prototype || target[prop] instanceof Date)
                     return target[prop];
-                const descriptor = Object.getOwnPropertyDescriptor(target, prop) || Object.getOwnPropertyDescriptor(Object.getPrototypeOf(target), prop);
+                const descriptor = Object.getOwnPropertyDescriptor(target, prop) || (Object.getPrototypeOf(target) && Object.getOwnPropertyDescriptor(Object.getPrototypeOf(target), prop));
                 if (descriptor && 'get' in descriptor) {
                     const value = descriptor.get.call(proxy);
                     if (proxifier.tree.master.options.devmode &&
