diff --git a/plugin.js b/plugin.js
index edfe7d2acd4f5c1ae6f07eae60c3e3eabf815274..835c3932a8637d31ea00269082f51c3818219292 100644
--- a/plugin.js
+++ b/plugin.js
@@ -340,12 +340,13 @@ function buildWorkletString(t, fun, closureVariables, name) {
     },
   });
 
+  const fnBody = fun.program.body[fun.program.body.length - 1]
   const workletFunction = t.functionExpression(
     t.identifier(name),
-    fun.program.body[0].expression.params,
+    fnBody.expression.params,
     prependClosureVariablesIfNecessary(
       closureVariables,
-      fun.program.body[0].expression.body
+      fnBody.expression.body
     )
   );
 
