diff --git a/node_modules/@o/recoil/dist/recoil.development.js b/node_modules/@o/recoil/dist/recoil.development.js
index a76bd94..cb87e12 100644
--- a/node_modules/@o/recoil/dist/recoil.development.js
+++ b/node_modules/@o/recoil/dist/recoil.development.js
@@ -588,9 +588,9 @@ function Batcher(props) {
 }
 
 {
-  if (!window.$recoilDebugStates) {
-    window.$recoilDebugStates = [];
-  }
+  // if (!window.$recoilDebugStates) {
+  //   window.$recoilDebugStates = [];
+  // }
 }
 
 function makeEmptyTreeState() {
@@ -678,7 +678,7 @@ function RecoilRoot({
 
 
     {
-      window.$recoilDebugStates.push(replaced); // TODO this shouldn't happen here because it's not batched
+      // window.$recoilDebugStates.push(replaced); // TODO this shouldn't happen here because it's not batched
     }
 
     storeState.nextTree = replaced;
