diff --git a/dist/node.js b/dist/node.js
index 05fd52c3d19eec3a16442ee40379a5901f75c809..11becb5204c53fcb447139dd686ddbfc8a3d37bf 100644
--- a/dist/node.js
+++ b/dist/node.js
@@ -2773,7 +2773,7 @@ function register(esbuildOptions = {}) {
     const format = inferPackageFormat(dir, filename);
     const {code: js, warnings, map: jsSourceMap} = _esbuild.transformSync.call(void 0, code, {
       sourcefile: filename,
-      sourcemap: "both",
+      sourcemap: false,
       loader: getLoader(filename),
       target: options.target,
       jsxFactory: options.jsxFactory
