diff --git a/node_modules/esbuild-register/dist/node.js b/node_modules/esbuild-register/dist/node.js
index 666b430..3b49248 100644
--- a/node_modules/esbuild-register/dist/node.js
+++ b/node_modules/esbuild-register/dist/node.js
@@ -64,7 +64,7 @@ function register(esbuildOptions = {}) {
         const options = getOptions(path.dirname(filename));
         const { code: js, warnings, map: jsSourceMap } = esbuild.transformSync(code, {
             sourcefile: filename,
-            sourcemap: 'both',
+            sourcemap: false,
             loader: getLoader(filename),
             target: options.target,
             jsxFactory: options.jsxFactory,
