diff --git a/node_modules/react-dom/cjs/react-dom.development.js b/node_modules/react-dom/cjs/react-dom.development.js
index d51615a..2744cea 100644
--- a/node_modules/react-dom/cjs/react-dom.development.js
+++ b/node_modules/react-dom/cjs/react-dom.development.js
@@ -10029,6 +10029,7 @@ var updatedAncestorInfo = function () {};
   var didWarn$1 = {};
 
   validateDOMNesting = function (childTag, childText, ancestorInfo) {
+    return
     ancestorInfo = ancestorInfo || emptyAncestorInfo;
     var parentInfo = ancestorInfo.current;
     var parentTag = parentInfo && parentInfo.tag;
