diff --git a/node_modules/react-dom/cjs/react-dom.development.js b/node_modules/react-dom/cjs/react-dom.development.js
index 9ed64ab..c46664a 100644
--- a/node_modules/react-dom/cjs/react-dom.development.js
+++ b/node_modules/react-dom/cjs/react-dom.development.js
@@ -10177,6 +10177,7 @@ var updatedAncestorInfo = function () {};
   var didWarn$1 = {};
 
   validateDOMNesting = function (childTag, childText, ancestorInfo) {
+    return
     ancestorInfo = ancestorInfo || emptyAncestorInfo;
     var parentInfo = ancestorInfo.current;
     var parentTag = parentInfo && parentInfo.tag;
diff --git a/node_modules/react-dom/umd/react-dom.development.js b/node_modules/react-dom/umd/react-dom.development.js
index 0d725f2..c79e69d 100644
--- a/node_modules/react-dom/umd/react-dom.development.js
+++ b/node_modules/react-dom/umd/react-dom.development.js
@@ -10210,6 +10210,7 @@
     var didWarn$1 = {};
 
     validateDOMNesting = function (childTag, childText, ancestorInfo) {
+      return
       ancestorInfo = ancestorInfo || emptyAncestorInfo;
       var parentInfo = ancestorInfo.current;
       var parentTag = parentInfo && parentInfo.tag;
